ogre2/src/Ogre2GpuRays.cc

```diff
@@ -465,6 +468,32 @@ void Ogre2GpuRays::Destroy()
         this->dataPtr->ogreCompositorNodeDef2nd);
     this->dataPtr->ogreCompositorWorkspaceDef2nd.clear();
   }
+
+  if (this->scene)
+  {
+    Ogre::SceneManager *ogreSceneManager = this->scene->OgreSceneManager();
+    if (ogreSceneManager == nullptr)
+    {
+      ignerr << "Scene manager cannot be obtained" << std::endl;
```

Comment:
```suggestion
      ignerr << "Scene manager cannot be obtained and removed" << std::endl;
```

Reply:
tweaked error msg to be more specific. [db722be](https://github.com/ignitionrobotics/ign-rendering/pull/617/commits/db722be385cfe8a9b8fd569be1c64a81060b5bf2)